ports/xmlsec/CMakeLists.txt

```
@@ -76,28 +74,25 @@ add_compile_definitions(HAVE_CTYPE_H)
 add_compile_definitions(HAVE_MALLOC_H)
 add_compile_definitions(HAVE_MEMORY_H)
 add_compile_definitions(XMLSEC_DEFAULT_CRYPTO="openssl")
-add_compile_definitions(XMLSEC_NO_GOST)
-add_compile_definitions(XMLSEC_NO_GOST2012)
 add_compile_definitions(UNICODE)
 add_compile_definitions(_UNICODE)
 add_compile_definitions(_MBCS)
 add_compile_definitions(_REENTRANT)
 
 set_target_properties(xmlsec1 xmlsec1-openssl PROPERTIES VERSION ${XMLSEC_VERSION_MAJOR}.${XMLSEC_VERSION_MINOR})
 
+set(XMLSEC_CORE_CFLAGS "")
 if(NOT BUILD_SHARED_LIBS)
-    set(XMLSEC_CORE_CFLAGS "-DLIBXML_STATIC -DLIBXSLT_STATIC -DXMLSEC_STATIC -DXMLSEC_NO_CRYPTO_DYNAMIC_LOADING")
-    set(XMLSEC_OPENSSL_CFLAGS ${XMLSEC_CORE_CFLAGS})
-else()
-    set(XMLSEC_CORE_CFLAGS "-DXMLSEC_DL_WIN32")
-    set(XMLSEC_OPENSSL_CFLAGS ${XMLSEC_CORE_CFLAGS})
+    set(XMLSEC_CORE_CFLAGS "-DLIBXML_STATIC -DXMLSEC_STATIC -DXMLSEC_NO_CRYPTO_DYNAMIC_LOADING")
+    target_compile_definitions(xmlsec1 PUBLIC LIBXML_STATIC XMLSEC_STATIC XMLSEC_NO_CRYPTO_DYNAMIC_LOADING)
```

```suggestion
    target_compile_definitions(xmlsec1 PUBLIC XMLSEC_STATIC XMLSEC_NO_CRYPTO_DYNAMIC_LOADING)
```
`XMLSEC_STATIC` depends on actual configuration of `BUILD_SHARED_LIBS` for libxml2, not on `BUILD_SHARED_LIBS` in xmlsec.

---

Fixed thank you